command/command.go

```diff
@@ -23,9 +23,15 @@ var version = "v1.0.0-rc.1"
 // empty context
 var nocontext = context.Background()
 
+func new() {
+	a = 20
+	fmt.Printf(a)
+}
+
 // Command parses the command line arguments and then executes a subcommand program.
 func Command() {
 	app := kingpin.New("drone", "drone aws runner")
+	fmt.Println(a)
```

Comment:
Remove 'fmt.Println(a)' as it is unnecessary and causes a compilation error due to 'a' being an undefined variable.